src/harfbuzz-config.cmake

```diff
@@ -0,0 +1,5 @@
+get_filename_component(_harfbuzz_rootdir ${CMAKE_CURRENT_LIST_DIR}/../../../ ABSOLUTE)
+
+set(HARFBUZZ_INSTALL_PREFIX ${_harfbuzz_rootdir})
+set(HARFBUZZ_INCLUDE_DIRS   ${HARFBUZZ_INSTALL_PREFIX}/include/harfbuzz)
+set(HARFBUZZ_LIBRARIES      -L${HARFBUZZ_INSTALL_PREFIX}/lib -lharfbuzz)
```

Comment:
Rather than variables, something like this might be better:

```cmake
add_library(harfbuzz::harfbuzz UNKNOWN IMPORTED)
set_target_properties(harfbuzz::harfbuzz PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_harfbuzz_rootdir}/include/harfbuzz"
  INTERFACE_LINK_LIBRARIES "${_harfbuzz_rootdir}/@libdir@/@harfbuzz_library_name@")
```

Where `libdir` is the same as in `Makefile.am` and `harfbuzz_library_name` is the name of the final library (`libharfbuzz.a` or `libharfbuzz.so.x.y.z`). The `UNKNOWN` could even be replaced with `SHARED` or `STATIC` if possible.

Reply:
Sorry, not `INTERFACE_LINK_LIBRARIES`, but `IMPORTED_LOCATION`.

Reply:
Thanks for your feedback, thats great!

I wonder also if you are interested on taking over this on a separate PR? If not I will of course continue try to apply you comment and experiencing on this but just saying :) Also any comment on our main cmake? For example I wasn't familiar with namespace concept so none of it exists but I like to improve it also :)

Reply:
Also should it be "HarfBuzz" or "harfbuzz", I don't to prefer which one where.

Reply:
The namespace should match what `find_package()` should be. The name of the files should also match, so `find_package(HarfBuzz)` should have `lib/cmake/HarfBuzz/HarfBuzz-config.cmake`.

Reply:
Thank you! Now it generates, 
```
add_library(harfbuzz::harfbuzz UNKNOWN IMPORTED)
set_target_properties(harfbuzz::harfbuzz PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "/usr/local/include/harfbuzz"
  IMPORTED_LOCATION "/usr/local/${exec_prefix}/lib/@harfbuzz_library_name@")
```
Not there yet but I am working on it :)